src/teb_local_planner_ros.cpp

```diff
@@ -903,6 +905,15 @@ void TebLocalPlannerROS::saturateVelocity(double& vx, double& vy, double& omega,
     vy *= ratio_y;
     omega *= ratio_omega;
   }
+
+  max_vel_linear = std::max(std::max(max_vel_x, max_vel_y), max_vel_linear);
+  double vel_linear = std::sqrt(vx*vx + vy*vy);
```

Comment:
```suggestion
  double vel_linear = std::hypot(vx, vy);
```
S: use `std:hypot` instead